domains/imazhar101.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "For my Portfolio",
+  "repo": "https://github.com/imazhar101/portfolio",
+  "owner": {
+    "username": "imazhar101",
+    "email": "imazhar101@gmail.com"
+  },
+  "record": {
+    "RECORDTYPE": "imazhar101.github.io"
```

```suggestion
    "CNAME": "imazhar101.github.io"
```
RECORDTYPE is not a valid record type.